CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 2.8.5 - 
+
+feat(Typescript): Migrated the SDK to TypeScript for better type safety and improved development experience.
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏We haven't migrated to Typescript yet. I think this is a little misleading. Lets just say that we are releasing typescript definitions for our consumers to use.